opencga-app/app/scripts/docker/opencga-app/Dockerfile

```diff
@@ -46,4 +46,4 @@ EXPOSE 8080
 EXPOSE 8443
 
 # Launch Tomcat
-CMD ["/opt/tomcat/bin/catalina.sh", "run"]
\ No newline at end of file
+CMD ["./opt/opencga/conf/opencga-env.sh", "&&", "./opt/tomcat/bin/catalina.sh", "run"]
```

Comment:
You will still need to source the `-env.sh` file. `export` makes vars in a shell avaialble to child shells not to parent shells. Using `source` or `.` will execute the script in the same shell, hence making the var available to the next call and any child shells should the script spawn them. Test below:

```bash
$ cat test.sh
#!/bin/bash
export FOO="bar"
$ ./test.sh
$ echo $FOO

$ . ./test.sh
$ echo $FOO
bar
```

Reply:
All fixed up - good spot